addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/AddOnJobUnitTest.java

```diff
@@ -190,6 +191,7 @@ void shouldReturnFileConfigData() {
     }
 
     @Test
+    @Disabled // Disabled which the updateAddons option is disabled.
```

Comment:
`@Disabled("The updateAddons option is disabled.")`

Reply:
Ah ha - didnt know you could do that...